templates/components/products/modals/writeReview.html

```diff
@@ -39,25 +39,25 @@ <h5 class="product-title">{{ product.title }}</h5>
 
                 <!-- Name -->
                 {{#if product.reviews.author}}
-                    {{> components/common/forms/text name="revfromname" label=(lang 'products.reviews.form_write.name') value=product.reviews.author}}
+                    {{> components/common/forms/text id="writeReview-name" name="revfromname" label=(lang 'products.reviews.form_write.name') value=product.reviews.author}}
```

Comment:
Can we just use the `name` attribute as the `id` value so it is consistent with what we have on other forms in cornerstone.